src/js/header.js

```diff
@@ -0,0 +1,33 @@
+import '../styles/components/header.scss';
+
+function hideShowMenu() {
+  const iconMenu = document.querySelector('.menu__icon');
+  const menuBody = document.querySelector('.menu__body');
+  const menuBodyTablet = document.querySelector('.menu__body-tablet');
+  if (iconMenu) {
+    iconMenu.addEventListener('click', function () {
```

Comment:
рекомендую использовать стрелочную функцию, так у тебя не будет предупреждения от eslint + код будет лаконичнее + консистентный относительно твой второй функции

```suggestion
    iconMenu.addEventListener('click', () => {
```